scripts/deploymentextensions/rundjangomigrations/install.sh

```diff
@@ -0,0 +1,224 @@
+repo_url=`get_github_url "$oxa_tools_public_github_account" "$oxa_tools_public_github_projectname"`
+sync_repo $repo_url $oxa_tools_public_github_projectbranch $oxa_tools_repository_path $access_token $oxa_tools_public_github_branchtag
+log "Repository sync is complete" " " 2
+
+# execute the installer remote
+if [[ $remote_mode == 0 ]];
+then
+    # at this point, we are on the jumpbox attempting to execute the installer on the remote target 
+    log "Copying scripts to target server '$target_server_ip'"
+
+    # copy the installer & the utilities files to the target server & ssh/execute the Operations
+    scp  -o "StrictHostKeyChecking=no" "${current_path}/install.sh" $target_user@$target_server_ip:~/
+    exit_on_error "Unable to copy installer script to '${target_server}' from '${HOSTNAME}' !" "${ERROR_DJANGO_MIGRATIONS_FAILED}" "${notification_email_subject}" "${cluster_admin_email}"
+
+    scp -o "StrictHostKeyChecking=no" "${current_path}/utilities.sh" $target_user@$target_server_ip:~/
+    exit_on_error "Unable to copy utilities to '${target_server}' from '${HOSTNAME}' !" "${ERROR_DJANGO_MIGRATIONS_FAILED}" "${notification_email_subject}" "${cluster_admin_email}"
+
+    # build the command for remote execution (basically: pass through all existing parameters)
+    repository_parameters="--oxatools-public-github-accountname ${oxa_tools_public_github_account} --oxatools-public-github-projectname ${oxa_tools_public_github_projectname} --oxatools-public-github-projectbranch ${oxa_tools_public_github_projectbranch} --oxatools-public-github-branchtag ${oxa_tools_public_github_branchtag} --oxatools-repository-path ${oxa_tools_repository_path}"
+    misc_parameters="--cluster-admin-email ${cluster_admin_email} --target-edx-system ${target_edx_system} --target-django-application ${target_django_application} --target-server-ip ${target_server_ip} --remote"
+    
+    if [[ $debug_mode == 1 ]];
+    then
+        misc_parameters="${misc_parameters} --debug"
+    fi
+
+    remote_command="sudo bash ~/install.sh ${repository_parameters} ${misc_parameters}"
+
+    # run the remote command
+    log "Executing '${remote_command}' against ${target_server_ip}"
+
+    ssh -o "StrictHostKeyChecking=no" "${target_user}@${target_server_ip}" "${remote_command}"
+    exit_on_error "Could not execute the installer on the remote target: ${target_server_ip} from '${HOSTNAME}' !" "${ERROR_DJANGO_MIGRATIONS_FAILED}" "${notification_email_subject}" "${cluster_admin_email}"
+
+    log "Completed Remote execution successfully"
+    exit
+fi
+
+#############################################
+# Main Operations
+# this should run on the target server
+#############################################
+
+log "Starting main execution (remote exection mode)"
+
+# Run migrations for the target django_application
+/edx/app/edxapp/venvs/edxapp/bin/python /edx/app/edxapp/edx-platform/manage.py ${target_edx_system} migrate ${target_django_application} --settings=aws --noinput
```

Comment:
should we restart edx services after the migration?

Reply:
Now that you mention it, yes we do need to.